src/js/webSerial.js

```diff
@@ -108,32 +119,46 @@ class WebSerial extends EventTarget {
 
     async disconnect() {
```

Comment:
Should this be
```suggestion
    async function disconnect() {
```

Reply:
It's method in a class so no ndeed for `function`.